v1.0/coordinateSpace.schema.json

```diff
@@ -0,0 +1,67 @@
+{
+  "_type": "https://openminds.ebrains.eu/sands/CoordinateSpace",
+  "description": "Structured information describing a coordinate space.",
+  "required": [
+    "alias",
+    "anatomicalOrientation",
+    "fullName",
+    "origin",
+    "releaseDate",
+    "unit",
+    "versionIdentifier"
+  ],
+  "properties": {
+    "alias": {
+      "type": "string",
+      "_instruction": "Enter the user-defined identifier for this coordinate space (e.g. a short name)."
+    },
+    "anatomicalOrientation": {
+      "_instruction": "Add the axes orientation denoted in standard anatomical terms of direction (in 2D or 3D).",
+      "_linkedTypes": [
+        "https://openminds.ebrains.eu/controlledTerms/Axis"
+      ]
+    },
+    "defaultImages": {
+      "type": "array",
+      "_instruction": "Add one or several images used as visual representation in this coordinate space.",
+      "_linkedTypes": [
+        "https://openminds.ebrains.eu/sands/Image"
+      ],
+      "minItems": 1,
+      "uniqueItems": true
+    },
+    "fullName": {
+      "type": "string",
+      "_instruction": "Enter a descriptive name for this coordinate space."
+    },
+    "origin": {
+      "type": "array",
+      "_instruction": "Enter the origin of the coordinate system (central point where axes intersect; 2D, with z = 0).",
+      "items": {
+        "type": "number"
+      },
+      "minItems": 3,
+      "maxItems": 3
+    },
+    "relatedOntology": {
+      "type": "string",
+      "_instruction": "Enter the identifier (IRI) of the related ontological term that best describes this coordinate space.",
+      "format": "iri"
+    },
+    "releaseDate": {
+      "type": "string",
+      "_instruction": "Enter the date of first broadcast/publication/release of this coordinate space.",
+      "format": "date"
+    },
+    "unit": {
+      "_instruction": "Add the unit that is used for this coordinate space.",
+      "_linkedTypes": [
+        "https://openminds.ebrains.eu/core/quantitativeValue.schema.json"
```

Comment:
this does not link to a quantitative Value (this is always number + unit). here it would point to the controlled term unitOfMeasurement (or at least that how it should be called... might be only unit at the moment)